QubitServer/src/main/java/org/labrad/qubits/config/AdcBaseConfig.scala

```diff
@@ -0,0 +1,41 @@
+package org.labrad.qubits.config
+
+import java.util.Map
+
+import org.labrad.data.Request
+
+abstract class AdcBaseConfig(channelName: String, builProperties: Map[String, Long]) {
+
+  /**
+   * number of clock cycles to delay
+   */
+  protected var startDelay: Int = -1
```

Comment:
AFAICT this is different but in fact correct, given, the checks we perform later...